.github/workflows/rust.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: dtolnay/rust-toolchain@1.73.0
+    - uses: dtolnay/rust-toolchain@1.75.0
```

Comment:
I'd probably deal with bumping this in a separate PR that fixes the new clippy lints.